ts/llama_cloud_services/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# llama-cloud-services
+
+## v0.1.0
+
+First release for `llama-cloud-services`.
```

Comment:
are we changing the npm package name? Could we defer that change? 
- The org scoping is nice in js ecosystem. It makes things more intuitive and findable, and is a strong convention already
- This is basically the same package as before. Seems like we should continue where the previous versions left off

Seems like barring a strong reason, changing package name and versions will just be a point of frustration for users. 

Besides, I kinda think `llama-cloud-services` is a bad name 🙉 🙈